README.rst

```diff
@@ -246,3 +246,113 @@ Kør nedenstående kommando for at autogenerere dokumentationen::
 
 Dokumentation kan nu findes ved at åbne filen
 ``/sti/til/mora/docs/out/index.html``.
+
+Licens og Copyright
+-------------------
+
+Copyright OS2 Offentlig Digitaliseringsfællesskab. 
+Dette værk er licenseret under MPL2.0, læs mere i LICENSE.
+
+Værket anvender følgende Open Source software-komponenter:
+
+* `Babel <http://babel.pocoo.org/>`_, BSD License
```

Comment:
Hvad er motivationen for dette? Kan vi ikke henvise til `requirements.txt` i stedet? Derudover skelner listen ikke mellem egentlige afhængigheder, og ting vi blot bruger til udvikling.